docs/intro/syntax-reference.md

```diff
@@ -66,13 +66,13 @@ Configuration is the third part of Mermaid, after deployment and syntax. It deal
 
 If you are interested in altering and customizing your Mermaid Diagrams, you will find the methods and values available for [Configuration](../config/setup/README.md) here. It includes themes.
 This section will introduce the different methods of configuring the behaviors and appearances of Mermaid Diagrams.
-The following are the most commonly used methods, and they are all tied to Mermaid [Deployment](./n00b-gettingStarted.md) methods.
+The following are the most commonly used methods, and they are all tied to Mermaid [Deployment](./getting-started.md) methods.
 
 ### Configuration Section in the [Live Editor](https://mermaid.live).
 
 Here you can edit certain values to change the behavior and appearance of the diagram.
 
-### [The initialize() call](https://mermaid-js.github.io/mermaid/#/n00b-gettingStarted?id=_3-calling-the-javascript-api),
+### [The initialize() call](https://mermaid-js.github.io/mermaid/#/getting-started?id=_3-calling-the-javascript-api),
```

Comment:
This URL can be fixed to avoid an extra redirect.

Reply:
Got it, I will. I thought there was some tricky behavior inside and decided not to alter it

Reply:
By the way, we can change all the links that point directly to mermaid-js.github.io

Reply:
Yup

Reply:
Fixed